Gemfile

```diff
@@ -17,6 +17,10 @@ gem 'figaro'
 
 gem 'graphql'
 
+gem 'sass-rails'
```

Comment:
These are the gems that allow us to use graphiql without the sprocket line we previously had in our application.rb.

Reply:
Nice work, so does that mean we re-comment in that line?

Reply:
You should be able to!